modules/client/examples/01-client/06-get-transfers.ts

```diff
@@ -1,36 +1,42 @@
 /* MARKDOWN
-### Loading DAO activity
+### Get transfers from DAO's activity
 
-Retrieves the list of asset transfers to and from the given DAO (by default,
-from ETH, DAI, USDC and USDT, on Mainnet)
+Gets the list of asset transfers to and from DAOs.
+If passed a `daoAddressOrEns`, will only retrieve transfers for that DAO. Otherwise, it returns for all DAOs.
+
+By default, retrieves ETH, DAI, USDC and USDT, on Mainnet).
```

Comment:
Not sure if this is true, @brickpop we retrieve all transfers that are made with the deposit or withdraw functions, not only ETH, DAI, USDC and UST no?

Reply:
This depends on subgraph, all ERC20 outgoing transfers should be monitored